types/hooks/screens/useSignedChannelListScreenHook.types.ts

```diff
@@ -1,11 +1,12 @@
 import {ChannelList} from '../../database/schema/ChannelList.types';
+import {ContactScreenPayload} from './useChatUtilsHook.types';
 
 interface UseSignedChannelListScreenHook {
   channels: ChannelList[];
   goToChatScreen: (channel: ChannelList) => void;
   goToPostDetailScreen: (channel: ChannelList) => void;
   goToCommunityScreen: (channel: ChannelList) => void;
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function signature for `goToContactScreen` has been updated to accept a parameter of type `ContactScreenPayload`. This is a good change as it allows the source of the navigation to be tracked. However, it's important to ensure that all calls to this function are updated accordingly throughout the codebase to prevent any potential runtime errors.
```diff
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->